recipes/slow-buffer-to-buffer-alloc-unsafe-slow/README.md

```diff
@@ -0,0 +1,58 @@
+# SlowBuffer to Buffer.allocUnsafeSlow Codemod
+
+This codemod migrates deprecated `SlowBuffer` usage to `Buffer.allocUnsafeSlow()` to handle Node.js DEP0030.
+
+## What it does
+
+This codemod transforms:
+
+1. `SlowBuffer` constructor calls to `Buffer.allocUnsafeSlow()`
+2. Direct `SlowBuffer` calls to `Buffer.allocUnsafeSlow()`
+3. Import/require statements to use `Buffer` instead of `SlowBuffer`
+
+## Examples
+
+### Before
+
+```javascript
+const SlowBuffer = require("buffer").SlowBuffer;
+
+// Using SlowBuffer constructor
+const buf1 = new SlowBuffer(1024);
+const buf2 = SlowBuffer(512);
+
+// Direct import
+const { SlowBuffer } = require("buffer");
+const buf3 = new SlowBuffer(256);
+```
+
+### After
+
+```javascript
+const Buffer = require("buffer").Buffer;
+
+// Using Buffer.allocUnsafeSlow()
+const buf1 = Buffer.allocUnsafeSlow(1024);
+const buf2 = Buffer.allocUnsafeSlow(512);
+
+// Direct import
+const { Buffer } = require("buffer");
+const buf3 = Buffer.allocUnsafeSlow(256);
+```
+
+### ESM Support
+
+```javascript
+// Before
+import { SlowBuffer } from "buffer";
+const buf = new SlowBuffer(1024);
+
+// After
+import { Buffer } from "buffer";
+const buf = Buffer.allocUnsafeSlow(1024);
+```
+
+## References
+
+- [Node.js DEP0030: SlowBuffer](https://nodejs.org/api/deprecations.html#DEP0030)
+- [nodejs/node#58220](https://github.com/nodejs/node/pull/58220)
```

Comment:
```suggestion
```

No needed because you are making the link on the introduction and the pr isn't really useful in codemod context.